BFF/v3/docker/ContainerizedIdentityServer/HostingExtensions.cs

```diff
@@ -0,0 +1,80 @@
+using Duende.IdentityServer;
+using ContainerizedIdentityServer;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Serilog;
+
+namespace ContainerizedIdentityServer;
+
+internal static class HostingExtensions
+{
+    public static WebApplication ConfigureServices(this WebApplicationBuilder builder)
+    {
+        builder.Services.AddRazorPages();
+
+        var isBuilder = builder.Services.AddIdentityServer(options =>
+            {
+                options.Events.RaiseErrorEvents = true;
+                options.Events.RaiseInformationEvents = true;
+                options.Events.RaiseFailureEvents = true;
+                options.Events.RaiseSuccessEvents = true;
+
+                // see https://docs.duendesoftware.com/identityserver/v6/fundamentals/resources/
+                options.EmitStaticAudienceClaim = true;
+                options.IssuerUri = "https://localhost:7051";
+                })
```

Comment:
```suggestion
                // see https://docs.duendesoftware.com/identityserver/fundamentals/resources#resources-isolation
                options.EmitStaticAudienceClaim = true;
                options.IssuerUri = "https://localhost:7051";
            })
```

(updated docs link + formatting)